plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -38,6 +38,7 @@ key_features:
   - "Add files to the block or allow lists"
 troubleshooting: "Isolate Endpoint fails with 500 error - This will happen if an isolation action (Isolate or Unisolate) is in progress on the selected endpoint. Wait a few minutes and try again."
 version_history:
+  - "4.0.2 - `Monitor Alerts` - Add unit tests"
```

Comment:
Would it be better to change this to something nice and generic, e.g. about bumping the SDK, and remove the part about unit tests? 

Having a line about adding unit tests in software release notes seems a bit weird to me

Reply:
This would be typical in a plugin update. I'm open to suggestions on this, since the bulk of this PR is unit tests for the task